Flutter/datagrid/columns-sizing.md

```diff
@@ -89,7 +89,7 @@ Widget build(BuildContext context) {
 > **NOTE**  
     The `GridColumn.columnWidthMode` takes higher priority than the `SfDataGrid.columnWidthMode`.
 
-![columns filled based on view port size in flutter datagrid](images/autofit-columns/flutter-datagrid-fill-columns.png)
+![columns filled based on view port size in Flutter Datagrid](images/autofit-columns/flutter-datagrid-fill-columns.png)
```

Comment:
Use proper control name casing in all places.

Reply:
Changed